src/site/stages/build/plugins/check-cms-urls.js

```diff
@@ -26,7 +26,11 @@ function checkForCMSUrls(BUILD_OPTIONS) {
       );
       console.log(filesWithBadUrls.join('\n'));
 
-      if (BUILD_OPTIONS.buildtype === ENVIRONMENTS.VAGOVPROD) {
+      if (
+        [ENVIRONMENTS.VAGOVPROD, ENVIRONMENTS.VAGOVSTAGING].includes(
+          BUILD_OPTIONS.buildtype,
+        )
+      ) {
```

Comment:
I'm not sure about this one. This ultimately makes sure that there aren't any URLs of the internal CMS `prod.cms.va.gov`, for example, in the Drupal content. They should have been updated to be the VA domain, or with the domain omitted. Maybe this should also fail on Staging? Up to you